opendevin/controller/command_manager.py

```diff
@@ -8,26 +8,23 @@
 
 class CommandManager:
     id: str
-    directory: str
     shell: Sandbox
 
     def __init__(
             self,
             sid: str,
-            directory: str,
             container_image: str | None = None,
     ):
-        self.directory = directory
         sandbox_type = config.get(ConfigType.SANDBOX_TYPE).lower()
         if sandbox_type == 'exec':
             self.shell = DockerExecBox(
-                sid=(sid or 'default'), workspace_dir=directory, container_image=container_image
+                sid=(sid or 'default'), container_image=container_image
             )
         elif sandbox_type == 'local':
-            self.shell = LocalBox(workspace_dir=directory)
+            self.shell = LocalBox()
```

Comment:
Does LocalBox need a change to deal with not being given a workspace directory?

Reply:
Yup! Good catch.

Do you have a good (non-scary) way to test locally? Maybe we should add a `__main__` to LocalBox so it's easier to test

Reply:
I tend to test inside docker. I'll take a look at this after the PR merges. It can wait.